tracer/src/Datadog.Trace/Tagging/KafkaTags.cs

```diff
@@ -41,6 +41,9 @@ public KafkaTags(string spanKind)
         [Tag(Trace.Tags.KafkaTombstone)]
         public string Tombstone { get; set; }
 
+        [Tag(Trace.Tags.KafkaConsumerGroup)]
```

Comment:
Looks like we should make the SpanMetadataRules more strict...I really would have liked the existing integration tests fail because the tag key `kafka.group` exists on some spans now, even though it's not specified in `SpanMetadataRules.cs`

Reply:
Yeah, I guess we don't check that that tags _don't_ exist currently. I agree we potentially want to add that functionality, though we also probably also want/need to make it easy to add global tags to all spans too. I added the optional `kafka.group` span in this PR anyway, and we can look into it down the line